preview/MsixCore/msixmgr/InstallUI.cpp

```diff
@@ -709,6 +767,13 @@ void UI::ButtonClicked()
                 break;
                 case InstallationStep::InstallationStepError:
                 {
+                    g_installing = false;
+                    ShowWindow(g_percentageTextHWnd, SW_HIDE);
+                    ShowWindow(g_staticPercentText, SW_HIDE);
+                    ShowWindow(g_progressHWnd, SW_HIDE);
+                    ShowWindow(g_checkboxHWnd, SW_HIDE);
+                    ShowWindow(g_CancelbuttonHWnd, SW_HIDE);
+                    m_displayErrorString = sender.GetTextStatus();

```

Comment:
>GetTextStatus [](start = 50, length = 13)

i think we still want to re-map the error code and not use the msixResponse, so that we can localize these strings.

Reply:
Modified the datatype of textStatus to UINT so that handlers can pass in the UINT value to the UI and this can then be fetched from the localized file.

Reply:
Converted back to std::wstring. Since we have to separate out the lib and the UI, UI returns strings in English so that another UI can consume them. Our UI compares HR's and returns localized strings.